open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb

```diff
@@ -0,0 +1,2166 @@
+{
```

Comment:
<p>This is a system error, not a block of code, remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
So,can I just delete it?